core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/columns/DataColumns.kt

```diff
@@ -28,4 +32,20 @@ class DataColumns {
         val col = listOf(URI.create("http://example.com"), null).toColumn("a")
         col.type().toString() shouldBe "java.net.URI?"
     }
+
+    @Test
+    fun `allow no nulls in frame columns`() {
+        // enable kotlin.dataframe.debug=true for this
```

Comment:
emmm, strange that we have different behaviour here..it was a goal just to discover weird things

Reply:
True, all add `if (BuildConfig.DEBUG)` to this specific check. We don't want to enforce users to enable debug mode for local tests.

Reply:
It's basically testing that if debug mode is enabled, it will catch this case